docs/release-notes/december-2023.md

```diff
@@ -0,0 +1,136 @@
+---
+slug: december-2023
+append_help_link: true
+hide_title: true
+description: >-
+    Release notes include the changes, fixes, and additions in specific versions of Semgrep.
+toc_max_heading_level: 3
+title: December 2023
+tags:
+  - Release notes
+---
+
+# December 2023 release notes
+
+## 🔧 OSS Engine
+
+* The following versions of the OSS Engine were released in December 2023:
+
+  * [<i class="fas fa-external-link fa-xs"></i>
+    1.52.0](https://github.com/returntocorp/semgrep/releases/tag/v1.52.0)
+  * [<i class="fas fa-external-link fa-xs"></i>
+    1.53.0](https://github.com/returntocorp/semgrep/releases/tag/v1.53.0)
+  * [<i class="fas fa-external-link fa-xs"></i>
+    1.54.0](https://github.com/returntocorp/semgrep/releases/tag/v1.54.0)
+  * [<i class="fas fa-external-link fa-xs"></i>
+    1.54.1](https://github.com/returntocorp/semgrep/releases/tag/v1.54.1)
+
+## 🌐 Cloud Platform
+
+### Added
+
+* Semgrep IDE integrations now cache information about the repository so that it
+  does not traverse the entire repository on every scan to determine if the
+  files are valid targets for scanning; this improves scan times.
+* Users can now ignore findings locally in Semgrep IDE Extensions. These changes
+  persist between restarts.
```

Comment:
Hmm... "persist between restarts" but what happens when they commit + open the PR? Can you ask the engineer who worked on this what was the intent? I think it may be Austin.

Reply:
Per Austin:

> So when a user triages it in the ide, nothing is reported back to the app. This is just a local ignore that will persist between sessions. Nothing on the commit or PR

Reply:
I'd suggest adding a note there stating this is a local ignore only. Something like "These changes persist between restarts, but are local ignores. Findings will still be detected when Semgrep scans your code, typically when opening a PR or MR.

(It's "typically" because a Semgrep scan, i.e. `semgrep scan` in the CLI will also generate findings.)